nyc_data/ppe/tests.py

```diff
@@ -36,35 +36,63 @@ def setUp(self) -> None:
 
     def test_rollup(self):
         rollup = aggregations.asset_rollup(
-            datetime.now() - timedelta(days=28), datetime.now()
+            datetime.strptime('2020-04-12', '%Y-%m-%d') - timedelta(days=28), datetime.strptime('2020-04-12', '%Y-%m-%d')
         )
         self.assertEqual(len(rollup), len(dc.Item))
         # demand of 20 = 5 in the last week * 4 weeks in the period
+        self.assertEqual(rollup[dc.Item.gown], AssetRollup(asset=dc.Item.gown, demand=17, sell=5))
+
+        # Turn of use of hospitalization projection
+        rollup = aggregations.asset_rollup(
+            datetime.strptime('2020-04-12', '%Y-%m-%d') - timedelta(days=28),
+            datetime.strptime('2020-04-12', '%Y-%m-%d'),
+            use_ihme_projection=False
+        )
         self.assertEqual(rollup[dc.Item.gown], AssetRollup(asset=dc.Item.gown, demand=20, sell=5))
 
         future_rollup = aggregations.asset_rollup(
-            datetime.now() - timedelta(days=30), datetime.now() + timedelta(days=30)
+            datetime.strptime('2020-04-12', '%Y-%m-%d') - timedelta(days=30), datetime.strptime('2020-04-12', '%Y-%m-%d') + timedelta(days=30)
+        )
+        self.assertEqual(
+            future_rollup[dc.Item.gown], AssetRollup(asset=dc.Item.gown, demand=33, sell=1005)
+        )
+
+        # Turn of use of hospitalization projection
+        future_rollup = aggregations.asset_rollup(
+            datetime.strptime('2020-04-12', '%Y-%m-%d') - timedelta(days=30),
+            datetime.strptime('2020-04-12', '%Y-%m-%d') + timedelta(days=30),
+            use_ihme_projection=False
         )
         self.assertEqual(
             future_rollup[dc.Item.gown], AssetRollup(asset=dc.Item.gown, demand=42, sell=1005)
         )
 
     def test_mayoral_rollup(self):
         rollup = aggregations.asset_rollup(
-            datetime.now() - timedelta(days=28), datetime.now(),
+            datetime.strptime('2020-04-12', '%Y-%m-%d') - timedelta(days=28), datetime.strptime('2020-04-12', '%Y-%m-%d'),
```

Comment:
1. Use the date constructor directly: `datetime.date(year=2019, month=4, day=13)`
2. Define this date once, name it "`today`" and use it everywhere :-)

Reply:
Fixed in 
a2f452c